Modelica_DeviceDrivers/Resources/Include/MDDRealtimeSynchronize.h

```diff
@@ -875,6 +875,8 @@ DllExport void MDD_RTSyncDestructor(void* rtSyncObj) {
     }
 }
 
+#include <stdint.h>
```

Comment:
That change seems either independent of the actual change or undesired.

Reply:
as above